predModels/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class pModelsConfig(AppConfig):
```

Comment:
Rename this to `PredictionModelsConfig` or something like that. The lower case p at front is odd